packages/boxel/addon/components/boxel/card-picker/index.css

```diff
@@ -15,6 +15,7 @@
 
 .boxel-card-picker__dropdown {
   position: static;
+  width: var(--boxel-card-picker-width);
```

Comment:
I agree that the ultra-narrowness needs to be addressed but maybe this makes the button look a bit strange when a card has already been chosen? (new on left)

<img width="1024" alt="Boxel Components 2022-09-27 09-49-38" src="https://user-images.githubusercontent.com/43280/192560439-9ad639c7-2e55-41c5-96be-478c700b0667.png">

Reply:
Yes, I agree that this ugliness is... ugly. I decided not to spend more time on it for now since it has yet to be integrated into our production apps or designs.

Reply:
ya makes sense!